packages/ruleset/test/binary-schemas.test.js

```diff
@@ -26,7 +26,7 @@ describe(`Spectral rule: ${ruleId}`, () => {
     it('Array of binary, non-JSON content', async () => {
       const testDocument = makeCopy(rootDocument);
 
-      testDocument.paths['/v1/drinks/menu'].put.requestBody = {
+      testDocument.paths['/v1/drinkmenu'].put.requestBody = {
```

Comment:
Had to update this existing testcase because of the change that was needed in root-document.js.